back_end/src/main/java/com/software/grey/security/SecurityConfig.java

```diff
@@ -66,11 +70,11 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                                 .requestMatchers(HttpMethod.PUT, VERIFY_REGISTRATION).permitAll()
                                 .requestMatchers(HttpMethod.GET, TEST).permitAll()
                                 .requestMatchers(HttpMethod.POST, POST + REPORT_POST)
-                                        .hasAnyAuthority(Role.ADMIN.name(), Role.MODERATOR.name())
+                                .hasAnyAuthority(Role.ADMIN.name(), Role.MODERATOR.name())
                                 .requestMatchers(HttpMethod.DELETE, POST + REPORT_POST + "/**")
-                                        .hasAnyAuthority(Role.MODERATOR.name(), Role.ADMIN.name())
+                                .hasAnyAuthority(Role.MODERATOR.name(), Role.ADMIN.name())
                                 .requestMatchers(HttpMethod.DELETE, POST + REMOVE_REPORTED_POST + "/**")
-                                        .hasAnyAuthority(Role.MODERATOR.name(), Role.ADMIN.name())
+                                .hasAnyAuthority(Role.MODERATOR.name(), Role.ADMIN.name())
                                 .anyRequest()
                                 .authenticated()
```

Comment:
```.hasAnyAuthority``` should have extra space before it, just like it was before. Please change it to how it previously was.